capstone-frontend/src/components/SearchBookModal.jsx

```diff
@@ -128,6 +109,19 @@ export class SearchBookModal extends Component {
     }
   }
 
+  clubSubmit = async (gbookID) => {
+    let updateJson = {
```

Comment:
nit: `const` (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)